docs/deploy-and-configure/installation/.pages

```diff
@@ -2,5 +2,7 @@ nav:
     - Installation: index.md
     - "Scenario: Single Node Cloud Installation": scenario-single-node-cloud-installation
     - "Scenario: Local Installation": scenario-local-installation
+    - "Scenario: RedHat Enterprise Linux 7": scenario-redhat-enterprise-linux-7
```

Comment:
Can we remove this entry or do you want to create this page as well?
<img width="674" height="421" alt="Screenshot 2025-11-03 at 17 17 42" src="https://github.com/user-attachments/assets/84de3caf-9107-4f7b-89ac-78ea5c95613a" />